### ISSUE TITLE
v2.1.13 -> v2.1.14 causes babel preset-typescript error

### ISSUE THREAD

**Issue by jamiechong** [LegendApp/legend-state#352]
I have no idea what might be causing this error in my Expo React Native app, but when I upgrade this package from 2.1.13 to 2.1.14, I get the following error. 

``` 
error: ../node_modules/react-native-gesture-handler/src/components/DrawerLayout.tsx: /Users/jamie/Sites/my-app/node_modules/react-native-gesture-handler/src/components/DrawerLayout.tsx: Cannot transform the imported binding "Animated" since it's also used in a type annotation. Please strip type annotations using @babel/preset-typescript or @babel/preset-flow.
  653 |           style={[styles.drawerContainer, drawerStyles, drawerContainerStyle]}>
  654 |           <View style={dynamicDrawerStyles}>
> 655 |             {this.props.renderNavigationView(this.openValue as Animated.Value)}
      |                                                                ^^^^^^^^
  656 |           </View>
  657 |         </Animated.View>
  658 |       </Animated.View>
```

It took me weeks to trace it down to this package and then it was pretty quick to isolate which version. 
The diff for your changes is here: https://github.com/LegendApp/legend-state/compare/v2.1.13...v2.1.14 and the only thing that seems remotely related is `src/babel/index.ts` https://github.com/LegendApp/legend-state/compare/v2.1.13...v2.1.14#diff-5c1bac14e0b4841976218a3e6cd252ed76a028d15a30a5deaa0e95f896b7d436

Do you guys have any thoughts on what might be the issue? Is there a way to work around this so I don't have to pin my Legendstate version to 2.1.13?

**Comment by jmeistrich** [comment]
Sorry about that @jamiechong! That seems to happen very rarely with some specific babel configurations. It was reported a while ago by v3 alpha testers and was fixed in there, and I hadn't realized it affected v2 as well. So I just cherry picked it in and released v2.1.5 which should fix it. Can you try that version and see if it works?

For what it's worth 2.1.14 didn't have any major changes other than some babel improvements (which broke in some rare configurations), and I don't expect any further updates to v2 because v3 is a massive change and will going to beta soon. So you could update to 2.1.15 or pin to 2.1.13 until v3 is released.

**Comment by jamiechong** [comment]
@jmeistrich  - no worries. I appreciate the fix you got out so quickly! Took me a while to finally get around to validating, but yes it looks like this resolves my problem. Thanks again!
